test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
Trying this out locally, and I noticed that my `hot_threads` call returned zero threads. Elasticsearch introduced the `ignore_idle_threads` parameter which defaults to `true`. So if the cluster is largely idle, there will be no hot threads to output. Setting the `ignore_idle_threads` param to `false` gives some output text to match against.

This param was added in Dec 2014 https://github.com/elastic/elasticsearch/commit/242e631e95a1b9240d3d84b16912c69bc63f031b